lib/fog/vsphere/requests/compute/list_clusters.rb

```diff
@@ -41,7 +41,7 @@ def cluster_attributes cluster, datacenter_name
 
         def cluster_path(cluster, datacenter_name)
           datacenter = find_raw_datacenter(datacenter_name)
-          cluster.pretty_path.gsub(/(#{datacenter.name}|#{datacenter.hostFolder.name})\//,'')
+          cluster.pretty_path.gsub(/(#{datacenter_name}|#{datacenter.hostFolder.name})\//,'')
         end
       end
```

Comment:
So you are saying that the object `datacenter`does return the full path but you only need the relative path which in this case is `datacenter_name`?